README.md

```diff
@@ -1,117 +1,98 @@
 # puppet_bolt_server
 
-Welcome to your new module. A short overview of the generated parts can be found
-in the [PDK documentation][1].
-
-The README template below provides a starting point with details about what
-information to include in your README.
+This module installs and configures Bolt to use a local PuppetDB and the PCP transport
 
 ## Table of Contents
 
 1. [Description](#description)
 1. [Setup - The basics of getting started with puppet_bolt_server](#setup)
     * [What puppet_bolt_server affects](#what-puppet_bolt_server-affects)
-    * [Setup requirements](#setup-requirements)
-    * [Beginning with puppet_bolt_server](#beginning-with-puppet_bolt_server)
+1. [Installation - Step by step guide](#installation)
 1. [Usage - Configuration options and additional functionality](#usage)
```

Comment:
we probably should also mention the `taskplan` module and its intended use with this setup. Let's also add a diagram showing how this works.

Reply:
Good idea! I added the diagram :D